safe-defaults/tmpfiles-disable.conf.in

```diff
@@ -5,6 +5,8 @@ r @SYSCONFDIR@/opt/chrome/policies/managed/safe-search.json
 r /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@/1
 r /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search/@FLATPAK_ARCH@
 r /run/flatpak/extension/org.chromium.Chromium.Policy.safe-search
+r /var/local/share/ca-certificates/Cisco_Umbrella_Root_CA.crt
+r /var/local/share/ca-certificates
```

Comment:
I don't think this directory should be removed since it's a general local data directory.

Reply:
ok